<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaSetting</class>
 <widget class="QWidget" name="ParaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbTestTarget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>测试指标</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaA">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaC">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUnit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>单位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQualim">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>合格限</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaClass">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>曲线类别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaD">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>参数D</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leParaA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leParaB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leParaC">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leQualim">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leParaD">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTestTarget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>总大肠菌群</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>耐热大肠菌群</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbUnit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MPN/100ml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MPN/L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CFU/mL</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbParaClass">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>地表水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地下水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>水源地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自来水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>城市污水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>垃圾渗透液</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>游泳池水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自定义1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自定义2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbR2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>相关系数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leR2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbParaClass_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>曲线1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>曲线10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
